src/services/widgets/widgets.svc.coffee

```diff
@@ -222,13 +220,11 @@ angular
 
           $http.put(ImpacRoutes.widgets.update(dashboard.id, widget.id), data).then(
             (success) ->
-              updatedWidget = success.data
-              angular.extend widget, updatedWidget
-              deferred.resolve(widget)
+              angular.extend widget, success.data
             (error) ->
-              $log.error("ImpacWidgetsSvc: cannot update widget: #{widget.id}")
-              deferred.reject(error)
-          )
+              angular.merge(widget, opts)
+              $log.warn("ImpacWidgetsSvc: unable to remotely save widget state (id: #{widget.id})")
+          ).finally( -> deferred.resolve(widget))
 
```

Comment:
Don't think we can do it this way: the consumer of the service sometimes waits for a rejection of the deferred object.
Ex: widget name edition - the name has to fallback to its previous value in case of failure